Adyen/Core/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift

```diff
@@ -235,7 +235,7 @@ open class AbstractPersonalInformationComponent: PaymentComponent, PresentableCo
             lastNameItem?.value = $0.lastName
         }
         shopperInformation.emailAddress.map { emailItem?.value = $0 }
-        shopperInformation.telephoneNumber.map { phoneItem?.value = $0 }
+        shopperInformation.phoneNumber.map { phoneItem?.value = $0.value }
```

Comment:
@neelSharma12 there was a warning - is this the right way to resolve it?

Reply:
What was the warning?

Reply:
`telephoneNumber` is deprecated

Reply:
This just updates the phone number but not the prefix - do you see a way to also update the prefix?